main_view.html

```diff
@@ -8,44 +8,324 @@
+                            <p class="framework_description">Frameworks </p>
+                            
+                            <ul class="framework_buttons">
+                                
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Bootstrap</button> </li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Ruby on rails</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">RSpec</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Capybara</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Selenium</button></li>
+                            </ul>
+                             
+                             </div>
+                           
+                            </div>
+                </div>
+
+                    <div class="skills_main">
+                   
+                            
+                            
+                        <div class="skills_section">
+                            <div class="skills_img"> <img src="images/middle_section/icon-skills.svg" alt=""></div>
+                        
+
+                            <div class="skills_button_action">
+                            <p class="skills_description">Skills</p>
+                            
+                            <ul class="skills_buttons">
+                                
+                                     <li class="tools"><button class="button_skills" value="Input Buttom" type="submit">Codekit</button> </li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">GitHub</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">Codepen</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">Gitlab</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom" type="submit">Terminal</button></li>
+                            </ul>
+                             
+                             </div>
+                           
+                            </div>
+                </div>
+
+
+
+
+                    <div class="about_main">
+                   
+                                            
+                        <div class="about_section">
+                            <h1 class="about_me">About me</h1>
+                            <h2 class="content"> I can help you build a product , feature or website Look through some of my work and experience! 
```

Comment:
- [ ] Please use `h2` tag instead of `h1` to follow the HTML semantic tags :blush:
- [ ] Please use `p` tag instead of `h2` tag for your content :blush: